charter.html

```diff
@@ -168,8 +168,7 @@ <h2>Scope</h2>
             state.
         </p>
         <p>
-          The Working Group may consider designs that allow user agents for the
-          same user — including non-browser agents, like Operating Systems — to
+          The Working Group may consider designs that allow browser or browser-like user agents (using web technologies) to
```

Comment:
Is there a need to call out "web technologies" here?  The scope I am thinking of ensures that attribution is possible for a mobile app, which doesn't necessarily use web technology.

Reply:
I think the concern is that the lack of precision could potentially end up resulting in our scope becoming overlarge. Is there a different phrasing you'd suggest or do you think the phrasing without that specific parenthetical is sufficient?

Reply:
Dropping the parenthetical would help, yes.

Reply:
I think it's hard to drop that parenthetical & simultaneously keep the scope contained. Though the wording is present earlier in the diff, so maybe there's a way this whole diff could be tweaked to make both Martin and me happy.

Reply:
@hober I fully support assuring that the scope is well contained. That said, I'm potentially not creative enough to what might be unintentionally included by removing the parenthetical (and I'm also unclear on what actually qualifies as "web technologies". Would apps that communicate over the internet (but not in a browser) count?)

Can you provide a couple examples of surfaces/systems which shouldn't be included, but (without the parenthetical) might cause friction in the future as someone views them as included in the scope of the charter?

Reply:
> (and I'm also unclear on what actually qualifies as "web technologies". Would apps that communicate over the internet (but not in a browser) count?)

No. The web is only one of the many things that happens over the internet.

> Can you provide a couple examples of surfaces/systems which shouldn't be included, but (without the parenthetical) might cause friction in the future as someone views them as included in the scope of the charter?

You can imagine a browser-like app that doesn't use web technologies. A Gopher browser, for instance, would be a browser-like app that doesn't use web technologies.

Reply:
@hober - I would specifically *want* our charter to encompass both apps and websites - as advertising is shown on both, and purchases happen in both, and connecting the two is critical for measurement.

I'd also like to make sure that to the extent there is any server-side aggregation (e.g. MPC helper parties) that this is also something that would be in-scope for the working group.

Are you *opposed* to the scope of the PAT-WG including apps and MPC helper parties?

Reply:
> @hober - I would specifically _want_ our charter to encompass both apps and websites - as advertising is shown on both, and purchases happen in both, and connecting the two is critical for measurement.
> 
> I'd also like to make sure that to the extent there is any server-side aggregation (e.g. MPC helper parties) that this is also something that would be in-scope for the working group.
> 
> Are you _opposed_ to the scope of the PAT-WG including apps and MPC helper parties?

I'm opposed to chartering a WG at W3C with a scope broader than W3C's, which is the web. "Some software that maybe sometimes sends data over the internet" is far too broad.

I suspect many or maybe even all aspects of MPC helper parties could be specced at W3C, though it sounds like the kind of thing we might want to do a W3C/IETF split for like we've done in a number of other cases (WebRTC comes to mind).